test/runnable/future.d

```diff
@@ -0,0 +1,43 @@
+/* PERMUTE_ARGS:
```

Comment:
Why is there a compilable and runnable test ? Shouldn't it be just runnable ? (But with the content of what is currently in compilable).